scapy/arch/windows/compatibility.py

```diff
@@ -23,7 +24,9 @@ def sndrcv(pks, pkt, timeout = 2, inter = 0, verbose=None, chainCC=0, retry=0, m
         
     if verbose is None:
         verbose = conf.verb
+    error.muteLogLoading(True) #Do not show the warning messages (already shown from arch.windows.__init__)
```

Comment:
This is to avoid the double warning message (it actually occures when winpcap is not installed)